docs/assets.md

```diff
@@ -11,12 +10,12 @@ that you can use in your demos.
 ## Install extra
```

Comment:
Let's remove this entire section (header, text, code block), as simply installing `supervision` is enough, and that is explained elsewhere.

I verified that links pointing to the section header won't cause a crash, and will show the page itself.